src/openrct2-ui/windows/EditorScenarioOptions.cpp

```diff
@@ -652,18 +654,9 @@ static void window_editor_scenario_options_financial_mousedown(rct_window* w, rc
         case WIDX_INTEREST_RATE_DECREASE:
             if (gBankLoanInterestRate > 0)
             {
-                if (gBankLoanInterestRate > 80)
-                {
-                    game_do_command(
-                        0, GAME_COMMAND_FLAG_APPLY, EDIT_SCENARIOOPTIONS_SETANNUALINTERESTRATE, 80,
-                        GAME_COMMAND_EDIT_SCENARIO_OPTIONS, 0, 0);
-                }
-                else
-                {
-                    game_do_command(
-                        0, GAME_COMMAND_FLAG_APPLY, EDIT_SCENARIOOPTIONS_SETANNUALINTERESTRATE, gBankLoanInterestRate - 1,
-                        GAME_COMMAND_EDIT_SCENARIO_OPTIONS, 0, 0);
-                }
+                auto interest = std::max(80, gBankLoanInterestRate - 1);
```

Comment:
Should be `std::min`. Otherwise it causes the _minimum_ interest rate to be 80%...